cookbook/integrations/kubernetes/kfmpi/mpi_mnist.py

```diff
@@ -68,7 +68,7 @@ def training_step(images, labels, first_batch, mnist_model, loss, opt):
     retries=3,
     cache=True,
     cache_version="0.1",
-    requests=Resources(cpu="1", mem="600Mi"),
+    requests=Resources(cpu="1", mem="2000Mi"),
     limits=Resources(cpu="2"),
```

Comment:
since pod is being killed due to OOM during runtime, do we need to increase `limits` as well?
```suggestion
    limits=Resources(cpu="2", mem="2000Mi"),
```

Reply:
If we didn't set memory in the limit, the admin will use the request value (2000Mi) in the limit. 2000 is enough in this example.